src/common/icons/lady-bug-solid-icon.tsx

```diff
@@ -0,0 +1,51 @@
+    '7.60938C3.02604 7.72917 3 7.85938 3 8C3 8.14062 3.02604 8.27083 3.07812 8.39062C3.13021 8.51042 3.20052 8.61719 3.28906 ' +
+    '8.71094C3.38281 8.79948 3.48958 8.86979 3.60938 8.92188C3.72917 8.97396 3.85938 9 4 9ZM6 14C6.27083 14 6.53385 13.9635 6.78906 ' +
+    '13.8906C7.04427 13.8125 7.28906 13.7057 7.52344 13.5703L6 9L4.47656 13.5703C4.71094 13.7057 4.95573 13.8125 5.21094 13.8906C5.46615 ' +
+    '13.9635 5.72917 14 6 14ZM7 8C7 8.14062 7.02604 8.27083 7.07812 8.39062C7.13021 8.51042 7.20052 8.61719 7.28906 8.71094C7.38281 ' +
+    '8.79948 7.48958 8.86979 7.60938 8.92188C7.72917 8.97396 7.85938 9 8 9C8.14062 9 8.27083 8.97396 8.39062 8.92188C8.51042 8.86979 ' +
+    '8.61458 8.79948 8.70312 8.71094C8.79688 8.61719 8.86979 8.51042 8.92188 8.39062C8.97396 8.27083 9 8.14062 9 8C9 7.85938 8.97396 ' +
+    '7.72917 8.92188 7.60938C8.86979 7.48958 8.79688 7.38542 8.70312 7.29688C8.61458 7.20312 8.51042 7.13021 8.39062 7.07812C8.27083 ' +
+    '7.02604 8.14062 7 8 7C7.85938 7 7.72917 7.02604 7.60938 7.07812C7.48958 7.13021 7.38281 7.20312 7.28906 7.29688C7.20052 7.38542 ' +
+    '7.13021 7.48958 7.07812 7.60938C7.02604 7.72917 7 7.85938 7 8ZM9 12C9.14062 12 9.27083 11.974 9.39062 11.9219C9.51042 11.8698 ' +
+    '9.61458 11.7995 9.70312 11.7109C9.79688 11.6172 9.86979 11.5104 9.92188 11.3906C9.97396 11.2708 10 11.1406 10 11C10 10.8594 9.97396 ' +
+    '10.7292 9.92188 10.6094C9.86979 10.4896 9.79688 10.3854 9.70312 10.2969C9.61458 10.2031 9.51042 10.1302 9.39062 10.0781C9.27083 ' +
+    '10.026 9.14062 10 9 10C8.85938 10 8.72917 10.026 8.60938 10.0781C8.48958 10.1302 8.38281 10.2031 8.28906 10.2969C8.20052 10.3854 ' +
+    '8.13021 10.4896 8.07812 10.6094C8.02604 10.7292 8 10.8594 8 11C8 11.1406 8.02604 11.2708 8.07812 11.3906C8.13021 11.5104 8.20052 ' +
+    '11.6172 8.28906 11.7109C8.38281 11.7995 8.48958 11.8698 8.60938 11.9219C8.72917 11.974 8.85938 12 9 12ZM3.5 3.5C3.5 3.125 3.57812 ' +
+    '2.77344 3.73438 2.44531L2.64844 1.35156L3.35156 0.648438L4.33594 1.63281C4.57031 1.42969 4.82812 1.27344 5.10938 1.16406C5.39062 ' +
+    '1.05469 5.6875 1 6 1C6.3125 1 6.60938 1.05469 6.89062 1.16406C7.17188 1.27344 7.42969 1.42969 7.66406 1.63281L8.64844 ' +
+    '0.648438L9.35156 1.35156L8.26562 2.44531C8.42188 2.77344 8.5 3.125 8.5 3.5C8.5 3.56771 8.49479 3.63542 8.48438 3.70312C8.47917 ' +
+    '3.76562 8.47135 3.82812 8.46094 3.89062C8.07031 3.72396 7.66667 3.59635 7.25 3.50781C6.83854 3.41927 6.42188 3.375 6 3.375C5.57812 ' +
+    '3.375 5.15885 3.41927 4.74219 3.50781C4.33073 3.59635 3.92969 3.72396 3.53906 3.89062C3.52865 3.82812 3.51823 3.76562 3.50781 ' +
+    '3.70312C3.5026 3.63542 3.5 3.56771 3.5 3.5Z';
+
+export const LadyBugSolidIcon = NamedSFC('LadyBugSolidIcon', () => (
+    <svg width="12" height="16" viewBox="0 0 12 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path d={d} fill="black" fillOpacity="0.9" />
```

Comment:
Should this color come from css?

Reply:
I am a little unsure on how can you put `fill` in a SVG icon from CSS. Just curious. I could be totally wrong here.

Reply:
Found it. @lisli1 : Just for reference: https://css-tricks.com/cascading-svg-fill-color/